### PR TITLE
LIHADOOP-17034 Create HadoopShell job type in hadoop DSL

### PR DESCRIPTION
**Pull request by nntnag17**

### CODE REVIEW

**Comment by johnyu0520** [comment]
Should wait for Gaurav's review, but LGTM

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ The following were contributed by Jonathan Hung. Thanks, Jonathan!
 * `TOOLS-67569 Block Hadoop Client Teams from checking in Hadoop Jars in their MP`
 
 The following were contributed by Anant Nag. Thanks, Anant!
+* `LIHADOOP-17034 Create HadoopShell job type in hadoop DSL`
```
Comment (convexquad) [inline CONTRIBUTORS.md]: hadoop DSL -> Hadoop DSL

File: VERSIONS.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ the License.
 Note that the LinkedIn build system occasionally requires that we skip a
 version bump, so you will see a few skipped version numbers in the list below.
 
+0.7.10
+
+* LIHADOOP-17034 Create HadoopShell job type in hadoop DSL
```
Comment (convexquad) [inline VERSIONS.md]: hadoop DSL -> Hadoop DSL

**Comment by convexquad** [comment]
I made a couple of minor comments. Thanks for another GREAT contribution to the Hadoop DSL and the Hadoop Plugin. +1 to commit as far as consistency with the Hadoop DSL language!

Can you ask Gaurav to review for HadoopShell functionality? If he likes it, you are +1 to commit this PR.

**Comment by logiclord** [comment]
LGTM. Thanks!!

**Comment by nntnag17** [comment]
Updated the PR after addressing comments from Alex.

**Pull request merged**